common/compositor/va/varenderer.cpp

```diff
@@ -269,15 +269,15 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   HwcRect<float> source_crop = state.layer_->GetSourceCrop();
   surface_region.x = source_crop.left;
   surface_region.y = source_crop.top;
-  surface_region.width = source_crop.right;
-  surface_region.height = source_crop.bottom;
+  surface_region.width = source_crop.right - source_crop.left;
```

Comment:
@XiaosongWei @guangxin can u review and see if something is wrong as to why we dont scale in case surface_region and out_region dont match here.

Reply:
lgtm